src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,26 @@
 import React from 'react';
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+interface MovieCards {
+  title: string;
+  description: string;
+  imdbUrl: string;
+  imgUrl: string;
+}
+
+type Props = {
+  movies: MovieCards[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies = [] }) => (
+  <ul>
+    <div className="movies">
+      {movies.map(movie => (
+        <li key={movie.title}>
+          <MovieCard {...movie} />
```

Comment:
It's not the best solution to pass props
If you'll have more that 100 props in future in movie object - they all will be passed to MovieCard
You need to pass all object as movie property in MovieCard props
And then get needed props from this object in MovieCard component